docs/api/api/api.txt

```diff
@@ -1847,6 +1847,76 @@ PUT /message/?project=<project>&package=<package>
 XmlBody :message
 XmlResult: message
 
+== Status Checks
+  WARNING: This API is currently in beta and unstable
+
+=== Repositories
+
+<project>: The name of a project, e.g. home:user1
+<repository>: The name of a repository of the project, e.g. openSUSE_Tumbleweed
+<uuid>: The build id you have received for the repository, e.g. 12345
```

Comment:
I would not name it uuid, since it is just a string and not following the uuid syntax

Reply:
Well that we would have to change not here but in the code. I'll open an issue.